tools/js-sdk-release-tools/src/llc/generateRLCInPipeline/generateRLCInPipeline.ts

```diff
@@ -212,7 +212,7 @@ export async function generateRLCInPipeline(options: {
         const packageName = packageJson.name;
         logger.info(`Start to generate some other files for '${packageName}' in '${packagePath}'.`);
         if (!options.skipGeneration) {
-            await modifyOrGenerateCiYml(options.sdkRepo, packagePath, packageName, false);
+            await modifyOrGenerateCiYml(options.sdkRepo, packagePath, packageName, packageName.includes("arm"));
```

Comment:
```suggestion
            await modifyOrGenerateCiYml(options.sdkRepo, packagePath, packageName, packageName.startsWith("@azure/arm-"));
```

not sure. maybe this is better? could you double confim please?
This is used to avoid regarding `xxxxarmyyy` as management plane SDK.

Reply:
this is for rlc mgmt packages like ```@azure-rest/arm-xxx```, so I think "includes("arm")" is just enough